frontend/src/assets/styles/card.scss

```diff
@@ -0,0 +1,50 @@
+@use 'sass:map';
+@use '@bcgov-nr/nr-theme/design-tokens/light-theme.scss' as lightTheme;
+
+.p-card .p-card-header {
```

Comment:
the style for card header and content is in the common repo previously, but I feel this is just for our FAM, so moved it to our repo.